src/rhtap/core/integration/git/providers/github.ts

```
@@ -1,19 +1,19 @@
 import { GithubClient } from '../../../../../../src/api/git/githubClient';
-import { GitHubClientFactory } from '../../../../../../src/api/git/githubClientFactory';
+import { GithubClient } from '../../../../../api/github';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'GithubClient'. Consider to delete it or rename it.

'GithubClient' is defined here:


(lint/suspicious/noRedeclare)